visidata/loaders/geojson.py

```diff
@@ -36,7 +39,7 @@ def iterload(self):
                 for prop in feature.get('properties', {}).keys():
                     prop = self.maybeClean(prop)
                     if prop not in self.colnames:
-                        c = Column(name=prop, getter=getter_factory(prop))
+                        c = GeoJSONColumn(name=prop, prop=prop)
```

Comment:
VisiData typically uses `expr` for things like this, so that it's consistently editable on the Columns Sheet.